packages/lab/stories/dropdown-next/dropdown-next.stories.tsx

```diff
@@ -0,0 +1,149 @@
+import { Story, ComponentMeta } from "@storybook/react";
+
+import { DropdownNext, DropdownNextProps } from "@salt-ds/lab";
+import { Button } from "@salt-ds/core";
+import { ArrowDownIcon, ArrowUpIcon } from "@salt-ds/icons";
+import { useState } from "react";
+
+export default {
+  title: "Lab/Dropdown Next",
+  component: DropdownNext,
+} as ComponentMeta<typeof DropdownNext>;
+
+const SimpleListExample = [
+  "Alabama",
+  "Alaska",
+  "Arizona",
+  "Arkansas",
+  "California",
+  "Connecticut",
+  "Delaware",
+  "Florida",
+  "Georgia",
+];
+
+const ListExample = [
```

Comment:
as much as I love the fruit 😍 it should probably be a more finance related example :P